connector/src/main/scala/com/microsoft/kusto/spark/datasink/KustoWriter.scala

```diff
@@ -92,23 +95,22 @@ object KustoWriter {
     }
     else {
       try {
-        rdd.foreachPartition { rows => ingestRowsIntoTempTbl(rows, batchIdIfExists, writeOptions.timeout, storageUri) }
+        rdd.foreachPartition { rows => ingestRowsIntoTempTbl(rows, batchIdIfExists, storageUri, acc) }
```

Comment:
It's better not to pass the storageUri.
You should pass a KustoAuthentication, and then each worker can resolve its own storage.
We will get more spread data between the storage accounts.

Reply:
We can do that in another pull request, if you'd like.